### ISSUE TITLE
[IA refresh] Content ticket (all pages except "Who we are")

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4771]
### Background:
This ticket is to make sure all content is in place for user testing purposes and launch.

We are currently updating 4 major sections of our site:
- Home page
- Who we are - separate ticket because this page's layout will be driven by content (#4668)
- What we do
- What we fund - https://github.com/mozilla/foundation.mozilla.org/issues/4816

### Stages of content:
Phase 1: Content for user testing (MVP)
Phase 2: Fill in the gaps: choose quotes, metrics we want to display on pages to celebrate our victories (e.g.: number of petitions signed and others), make sure the buttons and titles are consistent, choose the partners to be displayed in our home page and others.
Phase 3: Final and refined content: combination of results from user testing + review of the wireframe's current content.

### Phase 1: 
 **Status: currently blocked by executive feedback**
What we need from content in order to do user testing:
- [ ] exec team approval and copy the design team requested. Our test is around navigation, findability and clarity - having the information from exec team is important to make sure we are testing a prototype that has a language close to the final product.
- [ ] [Consider localization challenges](https://docs.google.com/document/d/1P1lCQUCooamJ-zO7Yh82-g7gxodH_0MfZrNh_8PcmBM/edit?usp=sharing) (as per Théo's feedback)
- [ ] "[novice user](https://redpen.io/tse32332f203796afb)" language assessment: is the language of the wireframes accessible from a novice user standpoint? If not, ask Content team to do tweaks.

### Phase 2: 
 **Status: not started yet**
- [ ] identify "missing pieces" of information on the wireframes and who would be the best person to provide the info. Also, identify final approvers for each section.
- [ ] request info from people identified in the previous step
- [ ] update the wireframes
- [ ] "[novice user](https://redpen.io/tse32332f203796afb)" language assessment: is the language of the wireframes accessible from a novice user standpoint? If not, ask Content team to do tweaks.
- [ ] send updated version to final approvers

### Phase 3:
**Status: current blocked by Phase 1 + dependent on results of user testing**
- [ ] integrate changes from exec feedback
- [ ] integrate changes from user testing
- [ ] having a person responsible for proofreading the whole content to make sure it aligns with our [novice audience's expectations](https://redpen.io/tse32332f203796afb) and editing inconsistencies (Content QA: Content team or Comms).
- [ ] content is ready to launch!

### Resources

- Feedback requested from exec team is described in details on Kristina's email entitled "Follow-up on website IA presentation"
- current content used in wireframes [is here](https://docs.google.com/document/d/19Y5hNmEU4wix_gCoSvmhMKUk2F1Dht6Nwf1JQAu0lsg/edit?usp=sharing)
- [Live IA refresh environment](https://ia-refresh.mofostaging.net/en/)

**Comment by taisdesouzalessa** [comment]
@kristinashu @beccaklam please let me know if this ticket makes sense to you and feel free to edit it if you see any gaps

**Comment by kristinashu** [comment]
Let's look at copy as we finalize the designs for each page, making sure to consider novice users, localization, and proof reading.
